basic/src/test/java/org/jboss/hal/testsuite/test/configuration/infinispan/CacheContainersTestCase.java

```diff
@@ -61,8 +62,8 @@ public void after() throws OperationException, IOException, TimeoutException, In
     }
 
     @AfterClass
-    public static void afterClass() {
-        IOUtils.closeQuietly(client);
+    public static void afterClass() throws InterruptedException, TimeoutException, IOException {
+        client.close();
```

Comment:
Just curious: why don't use `IOUtils.closeQuietly(client)` any more in this particular case?